### PR TITLE
Fix repo

### PR DESCRIPTION
**Pull request by smillims**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1506,11 +1506,19 @@ Thanks for the course and good luck to students!
...
+![Murka](assets/images/my-love-cat.jpg)
+
+---
+
 # Hello to everyone! :smiley_cat:
 My name is [Olga](https://github.com/ORybka) and I am very glad to join this community :grin:
 **Good luck to everyone!**
 And meet my cat who saw some things in this world :D
 
+---
```
Comment (miroslav-kolomiets) [inline README.md:1521]: It shouldn't be here.

File: README.md [hunk 2]
```diff
@@ -1535,4 +1543,4 @@ My name is [Yakiv](https://github.com/Iakow) and I am a future kottan. Glad to b
 My name is [Stas](https://github.com/Jigsaff)
 Good luck to everyone!
 
----
\ No newline at end of file
+---
```
Comment (miroslav-kolomiets) [inline README.md:1546]: It shouldn't be here too.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Hi! Good job! But you have to resolve conflicts and your code  affects other parts. See comments below.

File: README.md (hunk 2 above)
Comment (smillims) [reply README.md:1550]: Can you check all correct mistakes which i did?

**Comment by lenadgit** [comment]
How to resolve conflicts 
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line

**Comment by smillims** [comment]
Closed

**Pull request closed without merging**